tests/test_gitlab_project.py

```diff
@@ -0,0 +1,83 @@
+from http import HTTPStatus
+
+import responses
+
+from gitlab_project_exporter.gitlab_project import (
+    GitlabProject,
+    MirrorStatusCode,
+    RemoteMirrorStatus,
+)
+
+from .conftest import PROJECT_URL
```

Comment:
Same as in `test_api`

Reply:
fixed in https://github.com/app-sre/gitlab-project-exporter/commit/baf4ac06dd3c452d48f7310d4c34f14727c1f4c5